drake/systems/plants/IKoptions.h

```diff
@@ -51,7 +51,7 @@ class DRAKEIKOPTIONS_EXPORT IKoptions
     void setqd0(const Eigen::VectorXd &lb, const Eigen::VectorXd &ub);
     void setqdf(const Eigen::VectorXd &lb, const Eigen::VectorXd &ub);
     void setAdditionaltSamples(const Eigen::RowVectorXd &t_samples);
-    void updateRobot(const RigidBodyTree * robot);
+    void updateRobot(RigidBodyTree * new_robot);
```

Comment:
The header declared the method twice, once with const and once without. But the implementation does not have const, and Swig was getting confused by the duplicate (and incorrect) declaration.